c-sharp/Controllers/RootController.cs

```diff
@@ -1,32 +1,26 @@
-using System;
-using System.IO;
-using System.Collections.Generic;
-using System.Linq;
-using System.Threading.Tasks;
 using Microsoft.AspNetCore.Mvc;
-using Microsoft.Extensions.Logging;
 
 namespace c_sharp.Controllers
 {
-  [ApiController]
-  [Route("")]
-  public class RootController : ControllerBase
-  {
+    [ApiController]
+    [Route("")]
+    public class RootController : ControllerBase
+    {
 
-    private readonly ILogger<RootController> _logger;
+        private readonly ILogger<RootController> _logger;
 
-    public RootController(ILogger<RootController> logger)
-    {
-      _logger = logger;
-    }
+        public RootController(ILogger<RootController> logger)
```

Comment:
Just a quick question: why do we have a logger instance, but still use Console.WriteLine for our logging?